src/main/java/com/uid2/admin/vertx/service/OperatorKeyService.java

```diff
@@ -184,6 +187,18 @@ private void handleOperatorAdd(RoutingContext rc) {
                 return;
             }
 
+            Set<Role> roles = (!rc.queryParams().contains("roles")
+                    || RequestUtil.getRoles(rc.queryParam("roles").get(0)) == null)
+                        ? new HashSet<>()
+                        : RequestUtil.getRoles(rc.queryParam("roles").get(0));
+
+            if (rc.queryParams().contains("roles")
+                    && rc.queryParam("roles").get(0) != ""
+                    && roles.stream().allMatch(Objects::isNull)) {
```

Comment:
This one should be `anyMatch()` instead of `allMatch()`

Reply:
I am thinking if user is given multiple roles and there is one invalid role, should we throw error or ignore it?